fpp

```diff
@@ -23,6 +23,10 @@ BASEDIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 PYTHONCMD="python"
 NONINTERACTIVE=false
 
+if [ -z "$FPP_DIR" ]; then
+  FPP_DIR=~/.fpp
```

Comment:
Probably an overabundance of caution (this is actually my first PR ever). I will add another commit without the shell-script change!